mathjaxandroid/src/main/java/de/timfreiheit/mathjax/android/MathJaxView.java

```diff
@@ -119,12 +129,16 @@ public void onPageFinished(WebView view, String url) {
      * called when webView is ready with rendering LaTex
      */
     protected void rendered() {
-        handler.postDelayed(new Runnable() {
-            @Override
-            public void run() {
-                mWebView.setVisibility(View.VISIBLE);
-            }
-        }, 100);
+        if (onMathJaxRenderListener != null) {
```

Comment:
@timfreiheit Thanks for the feedback. 

So the idea here as well as below (where webview being made Invisible has a check on listener) is that - if the listener is not set, then the webview defaults to its current behaviour.

I'll update the code with moving callback to inside the handler.